packages/components/src/context-actions/ContextActions.test.tsx

```diff
@@ -4,6 +4,20 @@ import { TestUtils } from '@deephaven/utils';
 import ContextActionUtils from './ContextActionUtils';
 import ContextMenuRoot from './ContextMenuRoot';
 
+let spy: jest.SpyInstance;
+beforeEach(() => {
+  spy = jest.spyOn(window, 'requestAnimationFrame').mockImplementation(cb => {
```

Comment:
I would maybe add a comment pointing to the issue on github or an explanation that this causes inconsistent code coverage because the RAF call can quit early if the test runs super fast